### ISSUE TITLE
Do not remove empty fields

### ISSUE THREAD

**Issue by INCIBE-CERT** [python-rt/python-rt#70]
There are some fields like 'Cc' or 'AdminCc' that are not returned by get_ticket() when empty.
They are being removed by the `if lines` condition of __parse_response_dict()

**Comment by sim0nx** [comment]
I had another look at this and noticed that typing checks broke with your change.
As far as I can see __normalize_list is not supposed to return anything but str.
Indeed *Requestors*, *Cc* and *AdminCc* are handled different from other fields as they are supposed to be lists.

Could you please have a look at #72 and let me know if that fixes all the issues you have seen ?

If not, please let me know what other fields you see problems with that were treated differently before.

**Comment by INCIBE-CERT** [comment]
Hello @sim0nx Thanks for reviewing this.
Our bad, sorry about the typing checks.

We will need to look at #72 attentively. Hopefully tomorrow morning we will be able to.

Since __normalize_list() is only called on line 382/388, one could move that into an else branch and avoid the new lines at 275-277. However, I think it's probably cleaner to do it the opposite way: avoid the 385-387 change, and make instead __normalize_list() to return [] if the input is ''

**Comment by INCIBE-CERT** [comment]
This should be a simplified version of your #72.

**Comment by sim0nx** [comment]
Looks better indeed :-)... please also add a changelog to the CHANGES file (see my branch as possible inspiration)

Doing some testing I think we are still not there yet though.
Trying to create a new ticket with AdminCc or/and Cc set, does not actually set them with the proposed changes, I haven't look deeper than that yet.

```python
c = rt.Rt(url=RT_URL, default_login=RT_USER, default_password=RT_PASSWORD)
c.login(RT_USER, RT_PASSWORD)
new_ticket = {'Subject': 'Test subject',
              'Requestor': ['user@example.com'],
              'Text': 'test bla bla',
              'AdminCc': ['test1@example.com'],
              'Cc': ['test2@example.com'],
              }

tid = c.create_ticket('General', **new_ticket)
print(tid)
print(json.dumps(c.get_ticket(tid), indent=4))
```

**Comment by sim0nx** [comment]
FYI I have created a new PR (#75) based on your branch and added tests for the issue described in #70.

**Comment by sim0nx** [comment]
There is a new release with your fixes v2.2.2

Thanks for your contribution!
